src/providers/MeetingProvider/MeetingManager.ts

```diff
@@ -530,6 +540,7 @@ export class MeetingManager implements AudioVideoObserver {
       this.audioInputDevices.length
     ) {
       this.selectedAudioInputDevice = this.audioInputDevices[0].deviceId;
+      this.selectedAudioInputTransformDevice = this.selectedAudioInputDevice;
```

Comment:
Why do we change audio input transform device here? It is not in the description.